root/static/scripts/common/utility/doOrLogError.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * @flow strict
+ * Copyright (C) 2025 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import {captureException} from '@sentry/browser';
+
+export default function doOrLogError<T>(
```

Comment:
There are a lot of other places we could use this utility, but I didn't change them in this PR.